src/loki/status_action.cc

```diff
@@ -58,12 +60,19 @@ void loki_worker_t::status(Api& request) const {
 
   if (connectivity_map) {
     status->set_bbox(connectivity_map->to_geojson(2));
+    const bool has_transit_tiles =
+        connectivity_map->level_color_exists(TileHierarchy::GetTransitLevel().level);
+    status->set_has_transit_tiles(has_transit_tiles);
+  } else {
+    const static bool has_transit_tiles = !reader->GetTileSet(3).empty();
+    status->set_has_transit_tiles(has_transit_tiles);
   }
 
   status->set_has_tiles(static_cast<bool>(tile));
   status->set_has_admins(tile && tile->header()->admincount() > 0);
   status->set_has_timezones(tile && tile->node(0)->timezone() > 0);
   status->set_has_live_traffic(reader->HasLiveTraffic());
+  status->set_osm_changeset(tile ? tile->header()->dataset_id() : 0);
```

Comment:
we set the changeset here, but don't set it in the serializer's output JSON if it was 0